rb_ws/src/buggy/scripts/auton/trajectory.py

```diff
@@ -301,7 +301,7 @@ def get_unit_normal_by_index(self, index):
         return unit_normal
 
     def get_closest_index_on_path(
-        self, x, y, start_index=0, end_index=None, subsample_resolution=10000
+        self, x, y, start_index=0, end_index=None, subsample_resolution=1000
```

Comment:
We should note that this number is arbitrary and can be modified for speed purposes